core/src/main/scala/cats/UnorderedFoldable.scala

```diff
@@ -36,6 +36,12 @@ import simulacrum.{noop, typeclass}
   def unorderedFold[A: CommutativeMonoid](fa: F[A]): A =
     unorderedFoldMap(fa)(identity)
 
+  /**
+   * Tests if `F[A]` contains an `A` using the `Eq` instance for `A`
+   */
+  def contains[A](fa: F[A])(v: A)(implicit ev: Eq[A]): Boolean =
```

Comment:
Bikeshed: to curry or not to curry? It makes sense when the second argument is a lambda, less sure in this case.

Reply:
Yeah... That's the dilemma! :)
The concomitant  `exists` and `forall` methods are declared curried.. But they both takes labmdas (in contrast to `contains`).
I personally don't like currying unless it adds a value (for lambdas it usually does). So if Simulacrum is able to generate a correct syntax for uncurried `contains` I'd vote for making it uncurried.

BTW, are we determined to keep using Simulacrum-scalafix rules at all?

Reply:
@armanbilge @satorg since the name clashes with the stdlib one then I don't think it is too important or not if simulacrum can generate the right syntax.

Also, most people will be using the extension method instead of the method directly on the typeclass; and those who would use the typeclass directly probably expect the curried version; is there an example of a non-curried method? I can't recall any.

Reply:
That's fair... But if you prefer to keep the typeclass method corresponding to the syntax one, then you'd better to name it `contains_` (with underscore) rather than just `contains`, wouldn't you?

Reply:
I personally have no strict preference here. I prefer to avoid curried method _in general_ but realize it may not be applicable in cases like this one.

Reply:
> then you'd better to name it contains_ (with underscore) rather than just contains, wouldn't you?

Right, I actually just realized that was done with other methods like `filter_` I thought the `_` was just in the syntax.
I am fixing that right now.

Reply:
FWIW I don't think we should curry this. For example `as` is not curried.
https://github.com/typelevel/cats/blob/23d3c6c89bb0a1f77efd009804dd0b1d3ffb9d24/core/src/main/scala/cats/Functor.scala#L150

Reply:
Yeah, also most methods on `Traverse` are also not curried when not needed, `get` is the only exception. I will uncurry it.